docs/reference/field-types/README.md

```diff
@@ -0,0 +1,26 @@
+# Schema Field Type Reference
+
+See below for a list of all of the field types available in Apostrophe along with a brief description of their input. Click through to each for a complete description of its function, properties, and reference code to help you use them in your project.
+
+| Field | Input Description |
```

Comment:
You can review these descriptions even if the individual pages aren't ready yet.